src/graphql/models/Article.js

```diff
@@ -286,6 +287,18 @@ const Article = new GraphQLObjectType({
         });
       },
     },
+    articleType: {
+      type: ArticleTypeEnum,
```

Comment:
nit: I think we can
1. Add a resolver that resolves to `TEXT` if `articleType` does not exist for the document
2. Add `GraphQLNonNull` to its type so that client can know it cannot be `null`
3. Remove `LINE` from description because we may collect `article` from other channels as well